api/internal/plugins/execplugin/execplugin.go

```diff
@@ -140,6 +140,11 @@ func (p *ExecPlugin) Transform(rm resmap.ResMap) error {
 	return p.updateResMapValues(output, rm)
 }
 
+func (p *ExecPlugin) Validate(rm resmap.ResMap) error {
+	// Validate works exactly same with Transformer
```

Comment:
I'd rather defer introducing a new interface for now.

Once we make it, it's forever, and we're  not sure this  is a good idea.

In particular, the validator interface is identical to the  existing transformer interface - which suggests something is amiss. I suspect we'll move to three new interfaces more closely aligned with Config Functions, i.e. something like

```
 Generate(config) (ResourceList, error)
 Validate(config, ResourceList) error
 Transform(config, ResourceList) (ResourceList, error)
```
For now, the issue is only about adding a new field to run transformers -
specifically existing transformers - in a different context, as final step where 
they can make only limited changes.
Let's see if that gets used before formalizing a new interface.